draft-privacypass-rate-limit-tokens.md

```diff
@@ -474,6 +474,15 @@ Its ABNF is:
     Sec-Token-Request-Blind = sf-binary
 ~~~
 
+The "Sec-Token-Request-Key" is an Item Structured Header {{!RFC8941}}. Its
+value MUST be a Byte Sequence. This header is sent on Client-to-Attester
+requests ({{request-one}}), and contains a per-request public key.
+Its ABNF is:
+
+~~~
+    Sec-Token-Request-Key = sf-binary
```

Comment:
I think we need to add this to IANA considerations?

Reply:
Yep! Forgot to do that.